blocks/carousel/carousel.css

```
@@ -37,22 +37,33 @@
 }
 
 .carousel .carousel-slide > div {
+    flex-direction: column;
     font-family: var(--serif-font);
     font-weight: var(--font-weight-light);
     line-height: var(--line-height-120);
     letter-spacing: var(--letter-spacing-1-75);
     font-size: var(--font-size-28);
 }
 
-.carousel .carousel-slide > figure {
+/* Testimonial */
+.carousel.testimonial p {
+    margin-left: 1em;
+}
```

Add these font styles
```suggestion
  font-family: var(--sans-serif-font-regular);
  font-size: var(--font-size-16);
  letter-spacing: var(--letter-spacing-1);
}
```